astilectron.go

```diff
@@ -347,6 +347,14 @@ func (a *Astilectron) watchCmd(cmd *exec.Cmd) {
 	})
 }
 
+func (a *Astilectron) GetVersionAstilectron() string {
```

Comment:
Why do you need this ? 🤔

Reply:
Because I'm still using a foolish method to confirm the path in this [version](https://github.com/asticode/go-astilectron/pull/358/commits/ae08b36c5a8f4b2b68add1fd5808ad5e8201c78a#diff-f5c6513a93e0d3a0d10fb30654a66c3ab4b7f05ed5b4dd9dd10ef379a14496adR139-R141) 😅.

It's easy to get by `a.Paths().AstilectronDirectory()` and `a.Paths().ElectronDirectory()`